src/scss/take5-tutorials.scss

```diff
@@ -0,0 +1,91 @@
+@use 'base/breakpoints' as *;
```

Comment:
If this css file is specific to the hero, we could put this in a `hero` or `heros` directory and enhance from there.